libs/api/mocks/src/domains/license-service/factories.ts

```diff
@@ -37,12 +37,12 @@ const pkPassStatus = ['Available', 'NotAvailable', 'Unknown']
 
 export const genericLicense = factory<GenericLicense>({
   pkpass: () => faker.datatype.boolean(),
-  pkpassStatus: faker.random.arrayElement(
+  pkpassStatus: faker.helpers.arrayElement(
     pkPassStatus,
   ) as GenericUserLicensePkPassStatus,
   pkpassVerify: () => faker.datatype.boolean(),
   provider: () => ({
-    id: faker.random.arrayElement(providerArray) as GenericLicenseProviderId,
+    id: faker.helpers.arrayElement(providerArray) as GenericLicenseProviderId,
```

Comment:
In line 65 you inlined the const to avoid the cast, maybe do this here as well or maybe add ` as const` to the variable declaration.

Reply:
This seems reasonable. Fixed in 9ec483710c03908176f82eced823fcd5389c13d5.